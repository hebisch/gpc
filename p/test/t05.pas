program egwhile(output);
var
        n: integer;
        h: real;
begin
        n := 10;
        write(n);
        h := 0;
        while n > 0 do
        begin
                h := h + 1/n;
                n := n-1;
        end;
        writeln(h)
end.
