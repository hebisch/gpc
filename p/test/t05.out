10 2.928968253968254e+00
